src/main/java/org/ow2/proactive/catalog/rest/controller/BucketController.java

```diff
@@ -183,8 +184,10 @@ public List<BucketMetadata> list(
             @ApiParam(value = "The name of the user who owns the Bucket") @RequestParam(value = "owner", required = false) String ownerName,
             @ApiParam(value = "The kind(s) of objects that buckets must contain. Multiple kinds can be specified using comma separators") @RequestParam(value = "kind", required = false) Optional<String> kind,
             @ApiParam(value = "The Content-Type of objects that buckets must contain") @RequestParam(value = "contentType", required = false) Optional<String> contentType,
-            @ApiParam(value = "The name of objects that buckets must contain") @RequestParam(value = "objectName", required = false) Optional<String> objectName)
-            throws NotAuthenticatedException, AccessDeniedException {
+            @ApiParam(value = "The name of objects that buckets must contain") @RequestParam(value = "objectName", required = false) Optional<String> objectName,
+            @ApiParam(value = "Starts with bucket name (case insensitive)")
```

Comment:
after double checking, search by object names uses contains and not startsWith, so I think we should do the same for bucketName